### DISCUSSION TITLE
Google Chat Room for NOAA personnel

### DISCUSSION THREAD

**Discussion by darynwhite** [NOAA-PMEL/WRC-TUG#8] [Q&A] [answered]
To anyone that was at the meeting or interested in the Google Chat Room, please message me, Shaun, or Matt for us to add you to the room.

It turns out that the room is not visible to all as initially believed.

**Comment by mattcasari** [comment, marked as answer]
I did a PMEL ALL add, unless we want to keep adding people on an individual basis.

We will need to do that for anyone outside of PMEL.

**Comment by shaunwbell** [comment]
There is also a SLACK channel available in NOAA-PMEL workspace (#wrc-tech-users-group)

**Comment by darynwhite** [comment]
I feel like Slack needs pushing, and it will be used more. It seems the better option of the two options available to us.

**Reply by shaunwbell** [reply]
As of January 2022... Is this still the case?

Seems NOAA will be making SLACK an official solution (but at some additional expenses)

**Reply by darynwhite** [reply]
Those extra expenses are what will likely drive most of PMEL away; otherwise, it is the superior option, if only it were more integrated in some manner.
